.github/workflows/deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: deploy
+
+on: 
+  pull_request:
+    branches:
+      - dev
+      - master
+    paths:
+      - 'src/**'
+      # - 'test/**'
+      - 'build-libs.sh'
+      - 'Dockerfile'
+      - 'Makefile.essentiajs'
+      - 'rollup*.js'
+      - 'tsconfig.json'
+      - '!docs/**'
```

Comment:
We don't need `docs` and `examples` directory here, right?

Reply:
Exactly, the `/!docs/**` and `!examples/**` lines are excluding PRs that only change docs or examples, so it should be good as it is